routes/truckRoutes.js

```diff
@@ -1,3 +1,70 @@
-// connect to mongodb in this module as this is where you'll be making creat/read/delete calls to your database
+// connect to mongodb in this module as this is where you'll be making create/read/delete calls to your database
 // use 'mongodb://localhost/foodTruckAPI' for your mongoose connection string
 // remember this is a Node module
+
+var express = require('express');
+
+// the Router method returns an instance which can be mounted as middle-ware
+var router = express.Router();
+
+// '/trucks' routes
+app.route('/trucks')

```

Comment:
You want to use the router object you just created here instead of app. In app.js you're using app for your application instance.